auth/authenticate.js

```diff
@@ -1,8 +1,6 @@
 const jwt = require('jsonwebtoken');
 
-const jwtKey =
-  process.env.JWT_SECRET ||
-  'add a .env file to root of project with the JWT_SECRET variable';
+const jwtKey = require('./secret/jwtKey').jwtKey;
```

Comment:
Double check all your imports/exports and references to your jwtKey. In some places it is `jwtKey` and in some places it is `jwtSecret`. This is causing an error where it thinks the key has no value! In general, I think we've gotten a little too granular with the file/folder structure and simplifying it a little (like keeping the jwtKey in authenticate.js) could help with this.